src/main/java/com/example/travelservice/services/PersonalInfoService.java

```diff
@@ -11,4 +12,8 @@ public interface PersonalInfoService {
     PersonalInfo findById (Long id);
 
     Long findId(TravelerDto source);
+
+    Traveler updateEmail(Long id, String email);
```

Comment:
Why email? It should be just update, not only email might be changed, tomorrow you might add another property which is also updatable. If you want to define a business logic that some properties are immutable - fine, just check it in the code: if user is trying to update first name for example throw an exception that this field is not updatable.